chapters/projectManagement.typ

```diff
@@ -0,0 +1,208 @@
+
+#let numberOfWeeks = 15
+
+#let epic(
+  title: none,
+  startWeek: int,
+  backgroundColor: color,
+  foregroundColor: color,
+  inset: 6.5pt,
+  textSize: 0.9em,
+  itemStroke: white + 2pt,
+  items
+) = [
+  #let itemBox(width, content) = box(
+    width: 100% * width / (numberOfWeeks - (startWeek - 1)),
+    inset: inset,
+    fill: backgroundColor,
+    stroke: itemStroke,
+    text(
+      spacing: 100%,
+      fill: foregroundColor,
+      weight: "bold",
+      content
+    ),
+  )
+
+  #pad(left: 100% * (startWeek - 1) / numberOfWeeks, [
+    #set text(size: textSize)
+    #if title != none [
+      *#title* \
+    ]
+    #set text(spacing: 0%)
+    #for item in items {
+      itemBox(item.at(0), item.at(1))      
+    }
+  ])
+]
+
+#epic(
+  startWeek: 1,
+  backgroundColor: aqua,
+  foregroundColor: black,
+  inset: 4pt,
+  textSize: 1em,
+  itemStroke: none,
+  range(1, numberOfWeeks + 1)
+    .map(weekNum => (1, align(center, [w#str(weekNum)]))),
+)
+
+#epic(
+  title: "Documentation",
+  startWeek: 1,
+  backgroundColor: blue,
+  foregroundColor: white,
+  (
+    (2, "Setup"),
+    (8, "Ongoing documentation"),
+    (2, "Refinement"),
+    (2, [Abstract & MS #footnote[Management Summary]]),
+    (1, "Final"),
+  ),
+)
+
+#epic(
+  title: "1. Refactoring",
+  startWeek: 2,
+  backgroundColor: green,
+  foregroundColor: white,
+  (
+    (2, "Analysis"),
+    (2, "Implementation"),
+    (2, "Refinement"),
+    (2, "Pull Request"),
+  ),
+)
+
+#epic(
+  title: "2. Refactoring",
+  startWeek: 6,
+  backgroundColor: orange,
+  foregroundColor: white,
+  (
+    (2, "Analysis"),
+    (2, "Implementation"),
+    (2, "Refinement"),
+    (2, "Pull Request"),
+  ),
+)
+
+#line(length: 100%)
+*Actual Project Plan*
+
+#epic(
+  title: "Documentation",
+  startWeek: 1,
+  backgroundColor: blue,
+  foregroundColor: white,
+  (
+    (2, "Setup"),
+    (8, "Ongoing documentation"),
+    (2, [Abstract & MS #footnote[Management Summary]]), // TODO: use same footnote
+    (2, "Refinement"),
+    (1, "Final"),
+  ),
+)
+
+#epic(
+  title: "1. Refactoring",
+  startWeek: 2,
+  backgroundColor: green,
+  foregroundColor: white,
+  (
+    (1, "Analysis"),
+    (2, "Implementation"),
+    (1, "Refinement"),
+    (2, "Pull Request"),
+  ),
+)
+
+#epic(
+  title: "2. Refactoring",
+  startWeek: 6,
+  backgroundColor: orange,
+  foregroundColor: white,
+  (
+    (1, "Analysis"),
+    (4, "Implementation"),
+    (1, "Refinement"),
+    (2, "Pull Request"),
+  ),
+)
+#pagebreak()
+#set page(flipped: false)
+
+== Time Tracking <time-tracking>
+
+To monitor the working hours effectively, a Google Sheet was established where information about the tasks was meticulously recorded.
+Each record contains the date of execution, time spent, name of executioner, task category and a brief comment detailing the specific work performed during that time.
```

Comment:
```suggestion
Each record contains a date, the time spent, name of the person, task category and a brief comment detailing the specific work performed during that time.
```

Reply:
hans so ahpasst: "Each record contains a date, the amount of time spent, name of executioner, task category and a brief comment detailing the specific work performed during that time."

Reply:
executioner passi no ah, das tönt mer zu mörderisch